axes/helpers.py

```diff
@@ -148,6 +148,24 @@ def get_client_username(
     return request_data.get(settings.AXES_USERNAME_FORM_FIELD, None)
 
 
+def get_client_ip_address_default(request: HttpRequest) -> str:
+    try:
+        import ipware.ip
+    except ImportError as e:
+        raise AxesBackendImplementationRequired(
+            "Axes requires either configuring settings.AXES_CLIENT_IP_CALLABLE django-ipware package to be installed with e.g. pip install django-axes[ipware]."
+        ) from e
+
+    client_ip_address, _ = ipware.ip.get_client_ip(
+        request,
+        proxy_order=settings.AXES_PROXY_ORDER,
+        proxy_count=settings.AXES_PROXY_COUNT,
+        proxy_trusted_ips=settings.AXES_PROXY_TRUSTED_IPS,
+        request_header_order=settings.AXES_META_PRECEDENCE_ORDER,
```

Comment:
As you said in #1038, I think we should implement some logic to support old values and add warning that this values will be deprecated in 6.1 or 6.2 to ensure a smoother transition to new release

Reply:
Deprecation warnings added